blog_app/urls.py

```diff
@@ -7,4 +7,5 @@
     path('admin/', admin.site.urls),
     path('blogs/', include("blog.urls")),
     path('blogs/bloggers/', include("author.urls")),
+    path("__debug__/", include("debug_toolbar.urls")),
```

Comment:
Create a setting variable that allow us to enable or disable the debug toolbar. 
You can create a variable called `DEBUG_TOOLBAR` and if the value is set to 1, then only we need to enable this URL else not. 

Because the main purpose of the debug toolbar is to just profile the queries and code execution locally, so we don't need in the live server.

Reply:
Created DEBUG_TOOLBAR variable to enable and disable debug_toolbar.